microprofile-opentracing/src/main/resources/META-INF/beans.xml

```diff
@@ -0,0 +1,6 @@
+<beans xmlns="https://jakarta.ee/xml/ns/jakartaee"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_4_0.xsd"
+       version="4.0" bean-discovery-mode="all">
+    <!-- see https://issues.redhat.com/browse/WFLY-16947 -->
+</beans>
```

Comment:
"end of line" character is missing here, although each line should have its end according to the conventions ...

Reply:
fixed